vizro-core/src/vizro/actions/_callback_mapping/_callback_mapping_utils.py

```diff
@@ -97,17 +36,16 @@ def _get_inputs_of_controls(action_id: ModelID, control_type: ControlType) -> Li
 
 
 def _get_inputs_of_figure_interactions(
-    action_id: ModelID, action_function: Callable[[Any], Dict[str, Any]]
+    page: Page, action_function: Callable[[Any], Dict[str, Any]]
 ) -> List[Dict[str, State]]:
     """Gets list of States for selected chart interaction `action_name` of triggered page."""
```

Comment:
```suggestion
    """Gets list of States for selected chart interaction `action_function` of triggered `page`."""
```
If this is correct, then please check the other docstrings

Reply:
Should this even matter? Happy to follow a guideline here, but so far I don't think we have been consistently adding single quotation marks to model fields and Models in private methods. Shall we start doing that? Or shall we be less strict about it? I am fine either way